lte/gateway/c/core/BUILD.bazel

```diff
@@ -1138,6 +1138,10 @@ cc_binary(
     srcs = [
         "oai/oai_mme/oai_mme.c",
     ],
+    copts = [
+        "-DEMBEDDED_SGW",  # TODO(#11638): Feature flags fixups
+    ],
+    linkstatic = False,
```

Comment:
:'(

Reply:
I wonder why we haven't had this issue for other binaries that depend on glog.

Reply:
We experimented with a dummy C++ service in bazel that uses `glog` and static linking: as long as there are no logging calls in static variables the error does not occur. We were able to replicate the SEGV in the dummy service by calling logging in a static variable, see the example given in https://github.com/google/glog/issues/53.